app/routes/organizational-unit/show.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  renderTemplate(controller, model) {
+    if(model._internalModel.modelName === 'user'){
```

Comment:
This is a potential source of error in the future, since the `_internalModel` is not part of embers public API and might change in the future. However I don't think that there is another way to check for the class of the model.